arbeitszeit_flask/auth/routes.py

```diff
@@ -86,28 +87,16 @@ def signup_member(view: SignupMemberView):
 @auth.route("/member/confirm/<token>")
 @commit_changes
 @with_injection()
-def confirm_email_member(token, member_repository: MemberRepository):
+def confirm_email_member(
+    token, member_repository: MemberRepository, use_case: ConfirmMemberUseCase
```

Comment:
3 details:
- member_repository can get deleted
- token misses type hint
- nested function `def redirect_invalid_request()` is now unnecessary imo